modules/rostests/apitests/ntdll/CMakeLists.txt

```diff
@@ -7,6 +7,29 @@ include_directories($<TARGET_FILE_DIR:empty_dll>)
 include_directories(${REACTOS_SOURCE_DIR}/ntoskrnl/include)
 spec2def(ntdll_apitest.exe ntdll_apitest.spec)
 
+list(APPEND SOURCE_CRT_TESTS
+    ../crt/_snprintf.c
+    ../crt/_snwprintf.c
+    ../crt/_vscwprintf.c
+    ../crt/_vsnprintf.c
+    ../crt/_vsnwprintf.c
+    ../crt/mbstowcs.c
+    ../crt/mbtowc.c
+    ../crt/setjmp.c
+    ../crt/sprintf.c
+    ../crt/strcpy.c
+    ../crt/strlen.c
+    ../crt/strtoul.c
+    ../crt/wcstombs.c
+    ../crt/wcstoul.c
+    ../crt/wctomb.c
+)
+
+add_library(ntdll_crt_test_lib ${SOURCE_CRT_TESTS})
+target_compile_definitions(ntdll_crt_test_lib PRIVATE TEST_NTDLL)
```

Comment:
`TEST_NTDLL`: You are removing its 2 usages...
https://github.com/search?q=repo%3Areactos%2Freactos%20TEST_NTDLL&type=code

Reply:
So what? Maybe I'll add a new one later?

Reply:
So it looks like a "leftover". How to review what you might be doing later?